widgets/qt_pisces_main_window.py

```diff
@@ -4,12 +4,12 @@
 from PyQt5.QtGui import *
 import qdarkstyle
 import qt_textbox_and_slider
-import qt_custom_decorations
-import qt_view_laser_mode
+import qt_line_break
+import qt_live_mode_control
 import qt_left_window
 
 
-class MainWidgetWindow(QMainWindow):
+class PiscesMainWindow(QMainWindow):
```

Comment:
this package will not be specific to Pisces, intention is great but wrong place :)